packages/mobile/src/screens/transactions/components/item-details.tsx

```diff
@@ -36,13 +36,17 @@ const ItemDetail: FC<{
               {iconComponent}
             </View>
           )}
-          <Text
-            color={colors['text-title-login']}
-            variant="body1"
-            {...valueProps}
-          >
-            {value}
-          </Text>
+          {typeof value === 'string' ? (
+            <Text
+              color={colors['text-title-login']}
+              variant="body1"
+              {...valueProps}
+            >
+              {value}
+            </Text>
+          ) : (
+            value
```

Comment:
what is _value_ here ?

Reply:
value dùng để sử dụng cho nút copy, ví dụ như địa chỉ ví hiển thị bên ngoài là: orai12hz...qltx, nhưng copy thì cần value đầy đủ của địa chỉ ví đó.